src/Components/EnvironmentPicker/EnvironmentPicker.tsx

```diff
@@ -468,7 +468,7 @@ const EnvironmentPicker = ({
         resources: Array<IAzureResource | string>
     ) => {
         dispatch({
-            type: EnvironmentPickerActionType.HANDLE_CONTAINER_CHANGE,
+            type: EnvironmentPickerActionType.SET_CONTAINER_ITEMS,
             payload: {
```

Comment:
Still unable to clear values in the instance dropdown.
![cant delete](https://user-images.githubusercontent.com/57726991/186235977-0af620ec-a40a-4df6-b28d-6234e8de4757.gif)

Reply:
As in my other reply this is expected behavior: https://github.com/microsoft/iot-cardboard-js/pull/585#discussion_r944869971